pyvista/utilities/geometric_objects.py

```diff
@@ -661,7 +661,7 @@ def CircularArc(pointa, pointb, center, resolution=100, negative=False):
     angle = np.deg2rad(arc.GetAngle())
     arc = pyvista.wrap(arc.GetOutput())
     # Compute distance of every point along circular arc
-    center = np.array(center)
+    center = np.array(center).ravel()
```

Comment:
@akaszynski Thank you for fix. Could you explain why this change is need?

Reply:
Good question, this seems superfluous for a flat 3-length `center`.

Reply:
If for whatever reason someone puts in a center that's ``[[1, 2, 3]]``

Reply:
Reason why I added this was I'm getting a failure in one of my repositories and figured that it's better to simply fix this here since others might encounter the same issue.

Reply:
Slightly unusual approach :laughing: Harmless, I guess.